webapp/src/components/modals/create_event_form.css

```
@@ -0,0 +1,6 @@
+.create-event-form input[type="date"]::-webkit-calendar-picker-indicator {
```

Can we use a more specific class name, like `mscalendar-create-event-form`? This way it's definitely scoped to this plugin.

---

Also we generally prefer `scss` over `css` files, which allow you to do things like nesting:

```suggestion
.create-event-form {
    input[type="date"]::-webkit-calendar-picker-indicator {
        background: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'%3E%3Cpath d='M12 12H17V17H12V12M19 3H18V1H16V3H8V1H6V3H5C3.9 3 3 3.9 3 5V19C3 20.1 3.9 21 5 21H19C20.1 21 21 20.1 21 19V5C21 3.9 20.1 3 19 3M19 5V7H5V5H19M5 19V9H19V19H5Z' /%3E%3C/svg%3E") no-repeat;
        display: block;
        height: 18px;
        width: 18px;
    }
}
```

---

Thoughts about using the class name `mscalendar-create-event-form` instead? We usually try to scope css classes to a given plugin whenever possible

---

Oh yeah I got lost in thought about using `mscalendar` as class name and I forgot to put it while making other changes.